hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1252,6 +1253,43 @@ public function upgrade_1036() {
     return TRUE;
   }
 
+  /**
+   * Create new option group and values
```

Comment:
Do you think you can expand on the description a little bit, from a glance it should be clear which option group is being created